test/services/easypost.test.js

```diff
@@ -11,4 +12,43 @@ describe('EasyPost', function () {
       'Missing required parameter: API Key.',
     );
   });
+
+  it('will log the appropriate values when a request and response hooks are provided', async function () {
+    let requestConfig;
+    const requestHook = (response) => (requestConfig = response);
+    let responseConfig;
+    const responseHook = (response) => (responseConfig = response);
+
+    const client = new EasyPost(process.env.EASYPOST_TEST_API_KEY, {
```

Comment:
This unit test is throwing an error about an invalid API key. Is this the correct env var?